latest/docs-ref-conceptual/tutorial-azure-cli-server.yml

```diff
@@ -0,0 +1,246 @@
+### YamlMime:YamlDocument
+documentType: Tutorial
+title: Deploy a web server in Azure
+metadata:
+  title: Deploy a web server in Azure
+  description: Get started with the Azure CLI by deploying a VM with the NGINX web server.
+  audience: IT Pro
+  level: Beginner
+  displayType: two-column
+  interactive: azurecli
+items:
+- durationInMinutes: 1
+  content: >
+    In this getting started tutorial for the Azure CLI, you will learn how to create a Linux virtual machine,
+    install the NGINX web server, configure a data disk to share between resources, and configure the virtual
+    network firewall to allow HTTP and HTTPS traffic. You will do all of these tasks through the Azure CLI,
+    which can be used in-browser as part of this tutorial. Or you can [install the Azure CLI](install-azure-cli.md)
+    and follow along in your own terminal.
+
+- title: Create a Virtual Machine
+  durationInMinutes: 4
+  content: |
+      First, log in with your Azure subscription information. Follow any on-screen commands to get
```

Comment:
No need to starr this w/ First. Just "Log in to the Azure CLI" or something.

Reply:
This is something I debated, and I'm not sure how we want to be structuring our interactive tutorials in general. Either we can have "log in" be its own separate step in every tutorial, or we can integrate it as a part of the first task. I'm fine either way, we just need to make sure it's a consistent experience.

Reply:
keep in mind that if they are running with cloud shell in the other column they don't need to log in at all.